### ISSUE TITLE
wrong language name

### ISSUE THREAD

**Issue by ensag-dev** [dialect-app/po#6]
Hello,

I don't know the source of the language names in Occitan but they are all in French... and it's not even approaching =/

Some sources:
https://hosted.weblate.org/projects/iso-codes/iso-639-2/oc/
https://github.com/unicode-org/cldr/blob/main/seed/main/oc.xml
https://github.com/jitsi/jitsi-meet/blob/master/lang/languages-oc.json

I hope this can help!

**Comment by rafaelmardojai** [comment]
Hmm, so the Google translate site for `oc` (https://translate.google.com/?hl=oc) is in French right?

@fushinari, this is something I was concerned about our current approach. We eventually (or right now lol) will have Dialect UI translations for languages not available on Google. Did you research if using Weblate data was feasible?

**Comment by rafaelmardojai** [comment]
The **Unicode CLDR Project** seems interesting, but has different codes for the Chinese language variants.

**Comment by fushinari** [comment]
Yeah, maybe we need a better source, will look into the available options asap

**Comment by rafaelmardojai** [comment]
> Yeah, maybe we need a better source, will look into the available options asap

Nice. I will try to check that too.

**Comment by ensag-dev** [comment]
Hello,
Occitan is now present on the CLDR distributed files.
Best regards

**Comment by mufeedali** [comment]
Thanks for the update!

**Comment by rafaelmardojai** [comment]
@ensag-dev just checked the [cldr-json repo](https://github.com/unicode-org/cldr-json) and Occitan doesn’t includes the `languages.json` file.

**Comment by Mejans** [comment]
Hello!
Yesterday Google added Occitan to Google Translate :D
